daml-assistant/daml-helper/src/DamlHelper.hs

```diff
@@ -135,6 +137,7 @@ shouldReplaceExtension replaceExt dir =
 
 runJar :: FilePath -> [String] -> IO ()
 runJar jarPath remainingArguments = do
+    installSignalHandlers
```

Comment:
Why not put this in the first line of `main`?

Reply:
Good idea, I did that.